### ISSUE TITLE
Provide faculty accounts

### ISSUE THREAD

**Issue by philipmjohnson** [radgrad/radgrad#81]
Create/update accounts for the following faculty in the Galaxy deployment with the following interests:
* Kyungim Baek, research, visualization
* Edo Biagioni, research, networking, mobile
* Henri Casanova, research, high performance computing,
* Kim Binsted, research, artificial intelligence
* David Chin, research, artificial intelligence
* Martha Crosby, research, human-computer interaction
* Philip Johnson, research, software-engineering, civic-engagement, sustainability
* Jason Leigh, research, visualization
* Depeng Li, research, security, networking
* Lipyeow Lim, research, data mining, sustainability
* Carleton Moore, research, software engineering
* Dusko Pavlovic, research, security
* Guylaine Poisson, research, bioinformatics
* Scott Robertson, research, civic-engagement, human-computer interaction
* Peter-Michael Seidel, research, hardware, algorithms
* Nodari Sitchinava, research, algorithms
* Jan Stelovsky, research, human-computer interaction
* Susanne Still, research, machine learning
* Kazuo Sugihara, research, robotics
* Daniel Suthers, research, human-computer interaction

Note: the slug for a faculty member should be his email. So, the slug for Henri Casanova is henric, the slug for Philip is johnson, etc.

Next, in a branch called issue-81, please download the (square) headshot for each faculty member from http://www.ics.hawaii.edu/people/ and place into the public/images/people directory. Name it <slug>.png (or <slug>.jpg).  You will need to rename the image files, so Henri_Casanova.png becomes henric.png, etc.

Once that branch is merged into master and deployed to Galaxy, update the user's picture field to /public/images/people/<slug>.png.

The easiest way I've found to locate the file associated with a faculty's headshot is to inspect the page as follows:

![screen shot 2017-01-23 at 1 57 32 pm](https://cloud.githubusercontent.com/assets/290288/22228242/fc6a8e54-e173-11e6-96b7-40632317f8dc.png)

We will eventually want to associate opportunities with each faculty member, but just getting them defined is the first step.
